pages/api/v1/poll/[pollId]/results.ts

```diff
@@ -0,0 +1,290 @@
+import admin from 'firebase-admin';
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { Buffer } from 'buffer';
+import { getFirestore } from 'firebase-admin/firestore';
+import { cert } from 'firebase-admin/app';
+import { validate as isValidUUID } from 'uuid';
+import { createMessage } from '../../../../../utils/utils';
+import { Severity } from '../../../../../models/Enums';
+
+const X_USER_ID = 'x-user-id';
+
+type TransformedVote = string[];
+
+type CandidateCalculationStructure = {
```

Comment:
We can't use `TransformedVote` in place of `ResultVote` because we don't want to send _all_ of the individual votes to the front-end (for now, like we talked about) 

`TransformedVote` is just the raw votes and `ResultVote` is the minimized version of how many people votes with `X` candidate as the `firstChoice`